parser/page_test.go

```diff
@@ -63,8 +63,8 @@ func TestPage(t *testing.T) {
 			orgPageFrontMatter,
 			true,
 			map[string]interface{}{
-				"Title":      "Org Test 1",
-				"Categories": []string{"a", "b"},
+				"TITLE":      "Org Test 1",
+				"CATEGORIES": []string{"a", "b"},
```

Comment:
If case insensitivity was the motivation, you have gone from testing mix-case to upper case only.

But it's not important.

I forgot about this PR. I didn't want to pull it in late in the 0.19 game, but now is the time, but now there are some conflicts.